### DISCUSSION TITLE
Extension Requirement help

### DISCUSSION THREAD

**Discussion by AkashSCIENTIST** [nlohmann/json#2778] [Q&A] [answered]
Your project is very useful and helpful. Could you please help me to assign non-primitive types in JSON ...?

**Comment by nlohmann** [comment, marked as answer]
Did you check https://github.com/nlohmann/json#arbitrary-types-conversions ?

**Reply by nlohmann** [reply]
Do you need further assistance with this issue?

**Reply by AkashSCIENTIST** [reply]
As of now, I satisfied with your solution. Thanks, bro.
